## ¿Cuál será el resultado del primer console.log fuera del primer bucle (variable j).
## Resultado: No dara error por que "var" tiene un alcance global por lo tanto se encuentra disponible dentro y fuera de una funcion, asi que el valor de J fuera del bucle sera 3.


## ¿Qué va a ocurrir en el segundo console.lgo fuera del segundo bucle (variable k).
## Resultado: En este caso considero que dara error ya que "let" tiene un alcance de bloque, lo que lleva a que el valor de k se encuentra limitado al bloque en donde se esta usando.
